changelogs/unreleased/144-cruwe

```diff
@@ -0,0 +1 @@
+Fixes an issue where volumes meant to be filesystem datasets got created as zvols
```

Comment:
n_: support for making the parameters case insensitive in the storageclass?

Reply:
What does "n_" mean? Besides, added the general case. Which is more important to you and should accordingly come first? Or would you like me to drop the issue altogether?

Reply:
nit pick comment.

Reply:
@cruwe issue is fine. No worries there. Thanks for working on it.